src/app/App.js

```diff
@@ -47,4 +48,17 @@ export const App = ({ options }) => {
     modeHall.hidden = !modeHall.hidden;
     modeRules.hidden = !modeRules.hidden;
   }
+  
+  // dynamicaly set active game mode
+  gameModes.forEach(mode => {
+    mode.addEventListener('click', (e) => {
+      gameModes.forEach(mode => {
+        mode.classList.remove('menu__item--selected')
+      })
+      if (!e.target.classList.contains('menu__item--selected')) {
```

Comment:
to nie ma sensu jeżeli wczesniej usuwasz tę klasę ze wszystkich elementów, zostaw samą linijkę 59